test/integration/connect/test_connect.py

```diff
@@ -1578,3 +1599,263 @@ def test_output_format_tables_dialects(tt_cmd, tmpdir_with_cfg):
 
     # Stop the Instance.
     stop_app(tt_cmd, tmpdir, "test_app")
+
+
+def test_connect_to_single_instance_app_binary(tt_cmd):
+    if platform.system() == "Darwin":
+        pytest.skip("/set platform is unsupported by test")
+    tmpdir = tempfile.mkdtemp()
+    create_tt_config(tmpdir, "")
+    empty_file = "empty.lua"
+    # The test application file.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_single_app", "test_app.lua")
+    # The test file.
+    empty_file_path = os.path.join(os.path.dirname(__file__), "test_file", empty_file)
+    # Copy test data into temporary directory.
+    copy_data(tmpdir, [test_app_path, empty_file_path])
+
+    # Start an instance.
+    start_app(tt_cmd, tmpdir, "test_app", True)
+
+    # Check for start.
+    file = wait_file(os.path.join(tmpdir, "test_app", run_path, "test_app"),
+                     BINARY_PORT_NAME, [])
+    assert file != ""
+    file = wait_file(os.path.join(tmpdir, 'test_app'), 'configured', [])
+    assert file != ""
+
+    # Wait for instance to bootstrap
+    time.sleep(5)
```

Comment:
It is not good to sleep for 5 second for each test function. Isn't configured file existence is not enough?

Reply:
Changed, that was a problem with multi_inst_app not bootstrap in time, changed check, removed sleep.